test/Datadog.Trace.ClrProfiler.Managed.Tests/ScopeFactoryTests.cs

```diff
@@ -76,6 +76,21 @@ public void CleanUri_ResourceName(string uri, string method, string expected)
             }
         }
 
+        [Fact]
+        public void CreateOutboundHttpScope_Null_ResourceUri()
+        {
+            // Set up Tracer
+            var settings = new TracerSettings();
+            var writerMock = new Mock<IAgentWriter>();
+            var samplerMock = new Mock<ISampler>();
+            var tracer = new Tracer(settings, writerMock.Object, samplerMock.Object, scopeManager: null, statsd: null);
+
+            using (var automaticScope = ScopeFactory.CreateOutboundHttpScope(tracer, "GET", null, new IntegrationInfo((int)IntegrationIds.HttpMessageHandler), out _))
+            {
+                Assert.Equal(expected: "GET ",  actual: automaticScope.Span.ResourceName);
```

Comment:
Documents current behavior on `null`
We might want to look at changing this later.